libs/src/index.js

```diff
@@ -36,11 +36,12 @@ class AudiusLibs {
    */
   static configDiscoveryProvider (
     whitelist = null,
+    blacklist = null,
```

Comment:
could you add docstrings above and in libs/src/services/discoveryProvider/index.js to indicate this is a set